### PR TITLE
reef: mgr/dashboard: Dashboard RGW multisite configuration

### PR DESCRIPTION
**Pull request by aaSharma14**
Batch Backport of -

https://github.com/ceph/ceph/pull/49953 - https://tracker.ceph.com/issues/62387
https://github.com/ceph/ceph/pull/50529 - https://tracker.ceph.com/issues/59171
https://github.com/ceph/ceph/pull/50600 - https://tracker.ceph.com/issues/59190
https://github.com/ceph/ceph/pull/50557 - https://tracker.ceph.com/issues/59239
https://github.com/ceph/ceph/pull/50643 - https://tracker.ceph.com/issues/59328
https://github.com/ceph/ceph/pull/50806 - https://tracker.ceph.com/issues/62391
https://github.com/ceph/ceph/pull/51283 - https://tracker.ceph.com/issues/59328
https://github.com/ceph/ceph/pull/50706 - https://tracker.ceph.com/issues/61776
https://github.com/ceph/ceph/pull/52570 - https://tracker.ceph.com/issues/62110, https://tracker.ceph.com/issues/62109
https://github.com/ceph/ceph/pull/52644 - https://tracker.ceph.com/issues/62124

this backport was staged using ceph-backport.sh version 16.0.0.6848
find the latest version at https://github.com/ceph/ceph/blob/main/src/script/ceph-backport.sh

### CODE REVIEW

File: src/pybind/mgr/dashboard/requirements-test.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 pytest-cov
 pytest-instafail
 pyfakefs==4.5.0
-jsonschema==4.16.0
+jsonschema
```
Comment (nizamial09) [inline src/pybind/mgr/dashboard/requirements-test.txt:4]: this should be reverted!

File: src/pybind/mgr/dashboard/frontend/src/app/shared/constants/app.constants.ts [hunk 2]
```diff
@@ -137,11 +137,20 @@ export class ActionLabelsI18n {
...
 
   constructor() {
     /* Create a new item */
     this.CREATE = $localize`Create`;
 
+    this.EXPORT = $localize`Export`;
+
+    this.IMPORT = $localize`Import`;
+
+    this.MIGRATE = $localize`Migrate to Multi-Site`;
```
Comment (nizamial09) [inline src/pybind/mgr/dashboard/frontend/src/app/shared/constants/app.constants.ts:152]: i might have missed this in the original PR but this should have been just `Migrate`.

**Comment by nizamial09** [comment]
i see that this is also backporting the overview page. I feel its incomplete for a stable branch so what about waiting for the multisite sync status to be present as well? What do you think? @aaSharma14

**Comment by cloudbehl** [comment]
jenkins test dashboard

**Review by cloudbehl — APPROVED** [review]

**Pull request merged**